Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Keep version in-sync with .travis.yml and app/lib/shared/versions.dart
```

Comment:
Should we update .travis.yml too? And should we add the .mono_repo.yml files to this comment as well?

Reply:
Updated.